src/components/modals/CreateAntModal/CreateAntModal.tsx

```diff
@@ -349,12 +350,14 @@ function CreateAntModal({ show }: { show: boolean }) {
                       <div className="flex flex-column">
                         <Table
                           showHeader={false}
-                          onRow={(row: ManageAntRow) => ({
-                            className:
-                              row.attribute === editingField
-                                ? 'active-row'
-                                : '',
-                          })}
+                          onRow={(row: ManageAntRow) => {
+                            return {
+                              className:
+                                row.attribute === editingField
+                                  ? 'active-row'
+                                  : '',
+                            };
+                          }}
```

Comment:
Not sure if this was intended, but FWIW - wrapping an arrow function in () is shorthand for the explicit return statement.